packages/frontend/src/components/common/Filters/FilterGroupForm.tsx

```diff
@@ -0,0 +1,186 @@
+import { Button, Colors, HTMLSelect } from '@blueprintjs/core';
```

Comment:
This file can be ignored for now since we only show the SimplifiedFilterGroupFrom component. This component will be used once we support AND and OR operations. The UI will also change a bit.